### PR TITLE
Monorepo version maintainers minutes jul24

### PR DESCRIPTION
**Pull request by jfy133**
As ordered by @mashehu 

Note `npm run dev` didn't work... so I don't know if this is renering properly

### CODE REVIEW

**Comment by jfy133** [comment]
@mashehu @ewels this looks off, can you fix?

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+✍ Jon (@pinin4fjords) has been tasked formalising the discussions in proper guidelines, and will be posted on the nf-core website [guidelines page](https://nf-co.re/docs/guidelines/components/subworkflows) soon.
+
+## Daisy chains
+
+🌻 We discussed a recent pipeline proposal from Maxime (@maxulysse) which acts as one of the first nf-core forays into 'daisy-chaining' pipelines (i.e., importing pipelines into other pipelines).
+
+In this case the maintainers team collectively decided against making it an official pipeline at the moment.
+This was primarily due to concerns about a current lack of tooling and infrastructure in this context, particularly in how would documentation be 'merged' together, but also due to unresolved 'philosophical' discussions on what is the distinction between a subworkflow versus a workflow.
+
+🤔 However given the burning need by the community to have such functionality, we decided to set up the first (of two!) 'working groups', let by Rike (@FriederikeHanssen), to address this question. Feel free to join [#wg-meta-pipelines](https://nfcore.slack.com/archives/C07B5FK9GKA) join in.
```
Comment (FriederikeHanssen) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:46]:
```suggestion
🤔 However given the burning need by the community to have such functionality, we decided to set up the first (of two!) 'working groups', lead by Rike (@FriederikeHanssen), to address this question. Feel free to join [#wg-meta-pipelines](https://nfcore.slack.com/archives/C07B5FK9GKA) join in.
```

?

**Review by FriederikeHanssen — APPROVED** [review]

**Review by mahesh-panchal — APPROVED** [review]
Although I wasn't able to make this one, no issues from me.

**Comment by jfy133** [comment]
@mashehu @ewels feel free to merge when ready :)

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    src={nf_core_daisychain}
+    class="d-block m-auto"
+    width={750}
+    density={[1.5, 2]}
+    style="padding-bottom: 20px;"
+    alt="Picture of a daisy chain on a table with the nf-core apple logo on each of the foreground flowers."
+/>
+
+The 'Maintainers Minutes' aims to further give insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly Maintainer's team meetings.
```
Comment (SPPearce) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:26]:
```suggestion
by providing brief summaries of the monthly team meetings.
```

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Furthermore, members have said that knowing where to put, and then where to find existing test-data is hard to find due to insufficient documentation.
+In particular, the `delete_me/` folder has ballooned turning into a wild-west, contributing to the difficulties finding data.
+
+⛏ This working group will be led by Simon (@SPPearce), and if you want to give your input or get involved, you can join the [#wg-test-data-task-force](https://nfcore.slack.com/archives/C07B5FK9GKA) slack channel.
+
+## Ensuring quality
+
+☁️ Finally, there was a discussion about AWS megatests runs (runs of full-sized 'realistic' data executed on each pipeline release) regularly failing.
+
+It was brough up this often happens due to conflicts with Seqera's [Fusion](https://seqera.io/fusion/) file system, but more critically - these were only being discovered _after_ release.
```
Comment (SPPearce) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:63]:
```suggestion
It was brought up this often happens due to conflicts with Seqera's [Fusion](https://seqera.io/fusion/) file system, but more critically - these were only being discovered _after_ release.
```

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+title: 'Maintainers Minutes: July 2024'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2024-07-02T17:00:00+02:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-02/nf-core-daisychain.jpg'
+headerImageAlt: 'Picture of a daisy chain on a table with the nf-core apple logo on each of the foreground flowers.'
+authors:
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:7]:
```suggestion
embedHeaderImage: true
authors:
```

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    - 'jfy133'
+label:
+    - 'maintainers'
+---
+
+import nf_core_daisychain from '@assets/images/blog/maintainers-minutes-2024-07-02/nf-core-daisychain.jpg';
+import { Image } from 'astro:assets';
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:14]:
```suggestion
```

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+<Image
+    src={nf_core_daisychain}
+    class="d-block m-auto"
+    width={750}
+    density={[1.5, 2]}
+    style="padding-bottom: 20px;"
+    alt="Picture of a daisy chain on a table with the nf-core apple logo on each of the foreground flowers."
+/>
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-02.mdx:23]:
```suggestion
```
not needed when using `embedHeaderImage:true` in the frontmatter

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
and the dev preview command is now `npm run dev -w sites/main-site`

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Pull request merged**